### PR TITLE
`jsonb_pretty` function

### PR DESCRIPTION
**Pull request by anna-ahmed19**
Added th `jsonb_pretty` function under [Issue#4216](https://github.com/diesel-rs/diesel/issues/4216)

### CODE REVIEW

File: diesel/src/pg/expression/functions.rs [hunk 1]
```diff
@@ -1634,3 +1635,75 @@ define_sql_function! {
...
+    ///     .get_result::<String>(connection)?;
+    ///
+    /// assert_eq!(r#"null"#, result);
+    ///
+    /// let result = diesel::select(jsonb_pretty::<Jsonb, _>(json!({})))
+    ///     .get_result::<String>(connection)?;
+    ///
+    /// assert_eq!(r#"{
+    /// }"#, result);
+    /// #     Ok(())
```
Comment (guissalustiano) [inline diesel/src/pg/expression/functions.rs:1705]: Could you add a test case with a None input?

**Review by guissalustiano — CHANGES_REQUESTED** [review]
Thanks for opening the PR!

The changes look good, I only would like to see some tests with NULL input (I explain the reason [in yours other PR](https://github.com/diesel-rs/diesel/pull/4250#pullrequestreview-2296751838).

**Comment by anna-ahmed19** [comment]
Thank you for the suggestion.
I have added `MaybeNullableValue<Text>` and the `None` test.
Let me know if this is alright now.

**Review by guissalustiano — APPROVED** [review]
Thanks!
